packages/core/src/stories/Layouts.stories.js

```diff
@@ -68,7 +69,7 @@ storiesOf('Layout Examples', module)
   ))
 
 const Border = styled(Box)`
-  border: 1px solid ${(props) => props.theme.colors.lightGray};
+  border: 1px solid ${(props) => getPaletteColor('border.light')(props)};
```

Comment:
this can just be `border: 1px solid ${getPaletteColor('border.light')}`

Reply:
Updated!